hooks: Add "sqlalchemy.ext.baked" to sqlalchemy hook. (#5128)
pyinstaller/pyinstaller: news/5128.hooks.rst

> Updated a hook for sqlalchemy, by adding a hidden import.

Reviewer: This is a bit generic - please clarify what the hidden import is, and why it needed adding.

Reviewer 2: ```suggestion
Update ``sqlalchemy`` hook to support v1.3.19 and later,  by adding ``pysqlite2``, ``MySQLdb`` and ``psycopg2`` as hidden imports.
```

> Update ``sqlalchemy`` hook to support v1.3.19 and later,  by adding ``pysqlite2``, ``MySQLdb`` and ``psycopg2`` as hidden imports.

Reviewer: ```suggestion
Update ``sqlalchemy`` hook to support v1.3.19 and later,  by adding ``sqlalchemy.ext.baked`` as a hidden import
```

Reviewer 3: ```suggestion
Update ``sqlalchemy`` hook to support v1.3.19 and later.
```

Reviewer: Take whichever you like @RS357; it doesn't matter much.
